packages/world/src/VoxelCoord.sol

```diff
@@ -322,6 +322,15 @@ library VoxelCoordLib {
   function getPlayer(VoxelCoord memory coord) internal view returns (EntityId) {
     return ReversePlayerPosition._get(coord.x, coord.y, coord.z);
   }
+
+  function removePlayer(VoxelCoord memory coord) internal {
```

Comment:
it is a bit confusing to have both remove player and delete player imo

Reply:
how about in player utils: `createPlayer` and `deletePlayer`

And for `VoxelCoordLib`: `addPlayerToGrid`, `removePlayerFromGrid`

Reply:
hmm wdyt of having something like entityId.removePlayerPosition, that also takes care of schema positions and Position table?